### PR TITLE
[doc][multitenancy-manager] Uniform terminology

### PR DESCRIPTION
**Pull request by shvgn**
Signed-off-by: Eugene Shevchenko <evgeny.shevchenko@flant.com>

## Description

Clarifies the terms in Multitenancy Manager module

## Why do we need it, and what problem does it solve?

There is a little overwhelming use of synonyms in the documentation of the module. As we call out CRD Project and ProjectType, there should be projects and types, not environments and templates.

## Why do we need it in the patch release (if we do)?

To reduce possible confusion in for a reader.

## What is the expected result?

The module documentation is consistent and clean.

## Checklist
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in Kubernetes cluster

## Changelog entries

```changes
section: multitenancy-manager
type: chore
summary: Docs became more consistent
```

### CODE REVIEW

File: ee/modules/160-multitenancy-manager/crds/doc-ru-projects.yaml [hunk 1]
```diff
@@ -9,16 +9,14 @@ spec:
...
-                    Имя CR [ProjectType](cr.html#projecttype), который будет использоваться
-                    как шаблон для создания ресурсов окружения.
+                    Имя CR [ProjectType](cr.html#projecttype), который будет использоваться как шаблон для создания ресурсов проекта.
                 template:
                   description: |
-                    Значения для [темплейтов ресурсов](cr.html#projecttype-v1alpha1-spec-resourcestemplate) из шаблона окружения (CR [ProjectType](cr.html#projecttype))
-                    в формате helm values, которые соотносятся с
-                    OpenAPI-спецификацией (поле [openAPI](cr.html#projecttype-v1alpha1-spec-openapi) шаблона окружения).
+                    Входные параметры для [шаблона ресурсов](cr.html#projecttype-v1alpha1-spec-resourcestemplate) из типа проекта (CR [ProjectType](cr.html#projecttype)). Должны подходить под
+                    OpenAPI-спецификацию, указанной в типе проекта (поле [openAPI](cr.html#projecttype-v1alpha1-spec-openapi)).
```
Comment (z9r5) [inline ee/modules/160-multitenancy-manager/crds/doc-ru-projects.yaml:19]:
```suggestion
                    OpenAPI-спецификацию, указанную в типе проекта (поле [openAPI](cr.html#projecttype-v1alpha1-spec-openapi)).
```

File: ee/modules/160-multitenancy-manager/crds/doc-ru-projecttypes.yaml [hunk 2]
```diff
@@ -26,39 +25,38 @@ spec:
                     (параметр [resourcesTemplate](#projecttype-v1alpha1-spec-resourcestemplate)).
                 resourcesTemplate:
                   description: |
-                    Шаблоны ресурсов в формате `helm`, которые требуется создать при настройке нового окружения.
+                    Helm-шаблоны ресурсов, которые будут созданы при настройке нового проекта.
```
Comment (shvgn) [inline ee/modules/160-multitenancy-manager/crds/doc-ru-projecttypes.yaml:28]:
```suggestion
                    Helm-шаблон для ресурсов, которые будут созданы при настройке нового проекта.
```

File: ee/modules/160-multitenancy-manager/crds/doc-ru-projecttypes.yaml [hunk 3]
```diff
@@ -26,39 +25,38 @@ spec:
...
-                    - `{{ .projectName }}` — имя `Project` (окружения), для которого рендерится шаблон.
+                    - `{{ .projectName }}` — имя `Project` (проекта), для которого рендерится шаблон.
                     - `{{ .projectTypeName }}` — имя `ProjectType`.
-                    - `{{ .params }}` — словарь пользовательских значений, описанных в параметре [.spec.openAPI](cr.html#projecttype-v1alpha1-spec-openapi) и определенных в параметре [.spec.template](cr.html#project-v1alpha1-spec-template).
+                    - `{{ .params }}` — словарь входных параметров, описанных в поле [.spec.openAPI](cr.html#projecttype-v1alpha1-spec-openapi) и определенных в поле [.spec.template](cr.html#project-v1alpha1-spec-template).
 
-                    > **Внимание!** Указывать поля `.metadata.namespace` для объектов необязательно,
-                    > так как они автоматически проставятся из названия окружения (CR `Project`).
+                    > **Внимание!** Для объектов без поля `.metadata.namespace` оно будет автоматически присвоено
+                    > равным имени проекта (CR `Project`).
```
Comment (z9r5) [inline ee/modules/160-multitenancy-manager/crds/doc-ru-projecttypes.yaml:38]:
```suggestion
                    > Поле `.metadata.namespace` объекта по умолчанию соответствует имени проекта (CR `Project`), но вы можете явно задать другое значение.
```

File: ee/modules/160-multitenancy-manager/crds/projecttypes.yaml [hunk 4]
```diff
@@ -51,22 +50,21 @@ spec:
...
                     The following values are also available in templates:
-                    - `{{ .projectName }}` — the name of the `Project` (environment) for which the template is being rendered.
+                    - `{{ .projectName }}` — the name of the `Project` for which the template is being rendered.
                     - `{{ .projectTypeName }}` — the name of the `ProjectType`.
                     - `{{ .params }}` — a dictionary of custom values, described in the [.spec.openAPI](cr.html#projecttype-v1alpha1-spec-openapi) parameter and defined in the [.spec.template](cr.html#project-v1alpha1-spec-template) parameter.
 
-                    > **Note!** Specifying `.metadata.namespace` fields for objects is optional,
-                    > as they are automatically set with the name of the created environment (CR `Project`).
+                    > **Note!** Objects with missing `.metadata.namespace` will be assigned with one
+                    > set to the name of the created project (CR `Project`).
```
Comment (z9r5) [inline ee/modules/160-multitenancy-manager/crds/projecttypes.yaml:63]:
```suggestion
                    > By default, the object's `.metadata.namespace` parameter equals the project name (CR `Project`), but you can explicitly set a different value.
```

**Comment by z9r5** [comment]
The work has been done at least in the following PRs:
- https://github.com/deckhouse/deckhouse/pull/6544
- https://github.com/deckhouse/deckhouse/pull/6633
- https://github.com/deckhouse/deckhouse/pull/6864

**Pull request closed without merging**